composer.json

```diff
@@ -9,26 +9,26 @@
     "require": {
         "php": ">=5.5",
         "ext-curl": "*",
-        "guzzlehttp/guzzle": "~4.0",
-        "guzzlehttp/progress-subscriber": "~1.1",
-        "symfony/console": "~2.7.3",
-        "symfony/process": "~2.7.0",
-        "symfony/filesystem": "~2.7.0",
-        "symfony/finder": "~2.7.0",
-        "symfony/yaml": "~2.7.0",
-        "symfony/event-dispatcher": "~2.7.0",
-        "knplabs/github-api": "~1.3.1",
-        "ddd/slug": "~1.0.0",
-        "herrera-io/version": "~1.1.1",
-        "kherge/amend": "~3.0.0",
-        "ramsey/array_column": "~1.1.2",
-        "gushphp/gush-github-adapter": "~1.2.0",
-        "gushphp/gush-bitbucket-adapter": "~1.1.5",
-        "gushphp/gush-gitlab-adapter": "~1.1.4",
-        "gushphp/gush-jira-adapter": "~1.1.7"
+        "guzzlehttp/guzzle": "^4.0",
+        "guzzlehttp/progress-subscriber": "^1.1",
+        "symfony/console": "^2.7.3",
+        "symfony/process": "^2.7",
+        "symfony/filesystem": "^2.7",
+        "symfony/finder": "^2.7",
+        "symfony/yaml": "^2.7",
+        "symfony/event-dispatcher": "^2.7",
+        "knplabs/github-api": "^1.3.1",
+        "ddd/slug": "^1.0",
+        "herrera-io/version": "^1.1",
+        "kherge/amend": "^3.0",
+        "ramsey/array_column": "^1.1.2",
+        "gushphp/gush-github-adapter": "^1.2",
+        "gushphp/gush-bitbucket-adapter": "^1.1.5",
+        "gushphp/gush-gitlab-adapter": "^1.1.4",
+        "gushphp/gush-jira-adapter": "^1.1.7"
```

Comment:
Can I ask why do you not follow semver? I'm curious.

BTW, tests passes for this configuration, with all packages.